tests/Avalonia.Controls.UnitTests/Primitives/UniformGridTests.cs

```diff
@@ -140,5 +140,29 @@ public void Not_Visible_Children_Are_Ignored()
             // 2 * 2 grid
             Assert.Equal(new Size(2 * 50, 2 * 70), target.Bounds.Size);
         }
+
+        [Fact]
+        public void Grid_Expands_For_Row_And_Column_Spacing()
+        {
+            var target = new MyUniformGrid()
```

Comment:
What is MyUniformGrid?

Reply:
Oh sorry, I fixed it. (In the beginning I created a copy of UniformGrid called MyUniformGrid...)